src/main/java/com/meilisearch/sdk/IndexesHandler.java

```diff
@@ -37,12 +37,12 @@ String create(String uid) throws Exception, MeiliSearchApiException {
      * @throws Exception if an error occurs
      */
     String create(String uid, String primaryKey) throws Exception, MeiliSearchApiException {
-        JsonObject params = new JsonObject();
-        params.addProperty("uid", uid);
-        if (primaryKey != null) {
-            params.addProperty("primaryKey", primaryKey);
-        }
-        return meilisearchHttpRequest.post("/indexes", params.toString());
+        HashMap<String, Object> index = new HashMap<String, Object>();
```

Comment:
What about creating a type to make this easier and reducing the duplication in the updatePK method below?

Reply:
Yes, I also thought about it but we already have an index object. So I should create a `partialIndex` object or something and I wasn't sure about it ... No?